routes/cube_routes.js

```diff
@@ -911,8 +911,8 @@ router.get('/analysis/:id', async (req, res) => {
       });
       return cards;
     };
-    cube.cards = addDetails(cube.cards);
-    cube.maybe = addDetails(cube.maybe);
+    cube.cards = addDetails(cube.cards || []);
```

Comment:
Is there an easy way to set (the default value of) `.cards` and `.maybe` to `[]` for all cubes retrieved from `fineOne()`? That would prevent this sort of problem from happening in the future too.

Reply:
We have a default value in the Schema but that only works for new cubes